### DISCUSSION TITLE
Free tier credits vs 2 million revenue question

### DISCUSSION THREAD

**Discussion by imoby** [serverless/serverless#12709] [Q&A] [answered]
Hello

I have a small application that is not making much of anything at the moment in terms of revenue. I was curious if I went to v4 from v3, how does the credits vs the 2 million revenue come into picture.

The documentation honestly is not very helpful here. How does serverless determine how much revenue I am making per month? Let's say I have 2 separate aws accounts, each with a single stage setup (one for dev one for prod) does this mean because I am under the 2 million revenue mark that I can keep making instances (stages) and that I dont have to pay because I am not at that level?

**Comment by garethmcc** [comment, marked as answer]
@imoby We do not know if your organisation generates more than $2 million in annual revenue, unless of course you are a large enough company that is obvious or publish public data. However, most organisations over that threshold would prefer to remain legally compliant and are likely to pay for their use of Serverless Framework even if we can never really know for sure.

As for your use case, you can go ahead and have as many stages and services as you need. Not just 2. This can be across as many AWS accounts as you want.
